Gopkg.toml

```diff
@@ -75,7 +75,7 @@
 
 [[constraint]]
   name = "github.com/spf13/viper"
-  version = "1.0.0"
+  branch = "master"
```

Comment:
is this absolutely required? are we using something that is only available in `master`?

Reply:
@adnxn there were some prs merged regarding defaults being set up correctly so that when no environment variable and no flags were set for a variable the default can be expected. I recall not seeing that put in the the release, and you can see that a release hasn't happened in a while. I will retrace and double check if those PRs were pulled before or after the 1.0.0 cut, but i recall not being the case.

Reply:
@adnxn apparently i cant read dates, ha. Anyway i will revert to v1.0.0. and update with dep ensure. I double checked that defaults work and they seem to do that just fine.